sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1032,3 +1197,276 @@ def _from_generated(cls, score):
+            .format(self.name, repr(self.results))[:1024]
+
+
+class PiiEntitiesRecognitionTask(DictMixin):
+    """PiiEntitiesRecognitionTask encapsulates the parameters for starting a long-running PII Entities Recognition operation.
+
+    :ivar str name: An optional descriptive name for the task.
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str string_index_type: An optional string for specifying the method used to interpret string offsets.  
+        Defaults to Text Elements (Graphemes) according to Unicode v8.0.0. For additional information see 
+        https://aka.ms/text-analytics-offsets
+    :ivar str domain: An optional string to set the PII domain to include only a subset of the entity categories. Possible values include 'PHI' or None.
+    """
+
+    def __init__(self, **kwargs):
+        self.model_version = kwargs.get("model_version", "latest")
+        self.string_index_type = kwargs.get("string_index_type", "TextElements_v8")
+        self.domain = kwargs.get("domain", None)
+
+    def __repr__(self, **kwargs):
+        return "PiiEntitiesRecognitionTask(model_version={}, string_index_type={}, domain={})" \
+            .format(self.model_version, self.string_index_type, self.domain)[:1024]
+
+    def to_generated(self):
+        return _v3_2_preview_1_models.PiiTask(
+            enable=True,
+            parameters=_v3_2_preview_1_models.PiiTaskParameters(
+                model_version=self.model_version,
+                string_index_type=self.string_index_type,
+                domain=self.domain
+            )
+        )
+
+
+class PiiEntitiesRecognitionTaskResult(DictMixin):
+    """PiiEntitiesRecognitionTaskResult contains the results of a single PII Entities Recognition task, 
+        including additional task metadata.
+
+    :ivar str name: The name of the task.
+    :ivar results: The results of the analysis.
+    :vartype results: list[~azure.ai.textanalytics.RecognizePiiEntitiesResult]
+    """
+
+    def __init__(self, **kwargs):
+        self.name = kwargs.get("name", None)
+        self.results = kwargs.get("results", [])
+
+    def __repr__(self, **kwargs):
+        return "PiiEntitiesRecognitionTaskResult(name={}, results={})" \
+            .format(self.name, repr(self.results))[:1024]
+
+
+class EntityLinkingTask(DictMixin):
+    """EntityLinkingTask encapsulates the parameters for starting a long-running Entity Linking operation.
+
+    :ivar str name: An optional descriptive name for the task.
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str string_index_type: An optional string for specifying the method used to interpret string offsets.  
+        Defaults to Text Elements (Graphemes) according to Unicode v8.0.0. For additional information see 
+        https://aka.ms/text-analytics-offsets
+    """
+
+    def __init__(self, **kwargs):
+        self.model_version = kwargs.get("model_version", "latest")
```

Comment:
also missing `name` property. I think all Tasks are missing their `name` property, which leads me to believe that this is intentional. If so, would remove `name` entry from the docstrings

Reply:
I've removed this from the docstrings for the Task classes.